src/sql/workbench/api/node/extHostDataProtocol.ts

```diff
@@ -220,8 +220,8 @@ export class ExtHostDataProtocol extends ExtHostDataProtocolShape {
 		return this._resolveProvider<sqlops.QueryProvider>(handle).runQueryString(ownerUri, queryString);
 	}
 
-	$runQueryAndReturn(handle: number, ownerUri: string, queryString: string): Thenable<sqlops.SimpleExecuteResult> {
-		return this._resolveProvider<sqlops.QueryProvider>(handle).runQueryAndReturn(ownerUri, queryString);
+	$runQueryAndReturn(handle: number, ownerUri: string, queryString: string, isParse: boolean): Thenable<sqlops.SimpleExecuteResult> {
```

Comment:
were we making `isParse` it's own method?  If not, I'd prefer to see a QueryExecution options class that has all the configuration parameters instead of a bunch of individual options passed as additional parameters (I know there is only 1 extra one here, but I'm assuming more will be added later).

Reply:
@kburtram I just made a new request for it in LanguageService. I have PRs open on sqltoolsservice and dataprotocol-client to support that.